### PR TITLE
Added milestones to my profile

### PR DESCRIPTION
**Pull request by DevMukhtarr**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/1851"></a>

### CODE REVIEW

**Review by EmmaDawsonDev — APPROVED** [review]

**Review by krshkodes — APPROVED** [review]

File: public/data/devmukhtarr.json [hunk 1]
```diff
@@ -24,5 +24,23 @@
...
+      "description": "Wrote my first ever blog post which was about setting up nodejs and published it on hashnode",
+      "url": "https://mukhtar.hashnode.dev/setting-up-nodejs"
+    },
+    {
+      "title": "Completed Udacity Fullstack Nanodegree sponsored by ALX-T",
+      "date": "Aug 19, 2022",
+      "icon": "book",
+      "color": "blue",
+      "description": "I completed and learnt more about fullstack development with Udacity course",
+      "url": "https://graduation.udacity.com/confirm/GFVHADAL"
```
Comment (kumarsonsoff3) [inline public/data/devmukhtarr.json:43]: **Not sure what this url is about, is it the certificate of completion!** 
It's taking me directly to the sign in page at udacity.

I recommend using a public url, that everyone can see or have access to, or may be the course link!

Comment (DevMukhtarr) [reply public/data/devmukhtarr.json:43]: It is the the link to the certificate which is made public

Comment (kumarsonsoff3) [reply public/data/devmukhtarr.json:43]: Still it's redirecting me to the sign in page, not sure why!!

Comment (DevMukhtarr) [reply public/data/devmukhtarr.json:43]: Can there be milestone alone without link so I remove the certificate link

Comment (EmmaDawsonDev) [reply public/data/devmukhtarr.json:43]: Yes, you can remove the url key and it will still work.

Comment (DevMukhtarr) [reply public/data/devmukhtarr.json]: I have removed the URL

File: public/data/devmukhtarr.json [hunk 2]
```diff
@@ -24,5 +24,22 @@
...
       "icon": "linkedIn"
     }
+  ],
+  "milestones": [
+    {
+      "title": "Wrote my first blog post on hashnode",
+      "date": "Feb 6, 2021",
+      "icon": "hashnode",
+      "color": "blue",
+      "description": "Wrote my first ever blog post which was about setting up nodejs and published it on hashnode",
```
Comment (Panquesito7) [inline public/data/devmukhtarr.json:34]: To make it look more professional, what do you think if we capitalize the following words?
```suggestion
      "description": "Wrote my first ever blog post which was about setting up NodeJS and published it on Hashnode",
```

**Comment by DevMukhtarr** [comment]
Thanks @Panquesito7 that's fine

**Review by kumarsonsoff3 — APPROVED** [review]

**Pull request merged**